tests/dualtor/test_tor_ecn.py

```diff
@@ -350,8 +350,11 @@ def test_ecn_during_decap_on_active(
     exp_tos = encapsulated_packet[IP].payload[IP].tos
     exp_ecn = exp_tos & 3
     with stop_garp(ptfhost):
+        tor.shell("portstat -c")
         ptfadapter.dataplane.flush()
-        testutils.send(ptfadapter, int(ptf_t1_intf.strip("eth")), encapsulated_packet, count=10)
+        testutils.send(ptfadapter, int(ptf_t1_intf.strip("eth")), encapsulated_packet, count=1000)
+        time.sleep(1)
```

Comment:
The default timeout value is already 10s in verify_ecn_on_received_packet(), sleep 1s more here will help?